hack/functests.sh

```diff
@@ -64,6 +64,9 @@ function functest() {
     if [[ ${KUBEVIRT_PROVIDER} =~ .*(k8s-sriov).* ]] || [[ ${KUBEVIRT_SINGLE_STACK} == "true" ]]; then
         echo "Will skip test asserting the cluster is in dual-stack mode."
         KUBEVIRT_FUNC_TEST_SUITE_ARGS="-skip-dual-stack-test ${KUBEVIRT_FUNC_TEST_SUITE_ARGS}"
+    elif [[ ${KUBEVIRT_PROVIDER} =~ .*(ovn).* ]]; then
```

Comment:
@qinqon Can you please elaborate on the reason for this skipping in the commit msg?